src/Api/Models/Request/UpdateDomainsRequestModel.cs

```diff
@@ -1,13 +1,12 @@
 ﻿using System.Text.Json;
 using Bit.Core.Entities;
-using Bit.Core.Enums;
 
 namespace Bit.Api.Models.Request;
 
 public class UpdateDomainsRequestModel
 {
     public IEnumerable<IEnumerable<string>> EquivalentDomains { get; set; }
-    public IEnumerable<GlobalEquivalentDomainsType> ExcludedGlobalEquivalentDomains { get; set; }
+    public IEnumerable<int> ExcludedGlobalEquivalentDomains { get; set; }
```

Comment:
swagger should be able to build the enum here, I don't think this change is a good idea

Reply:
It can, however this causes the equivalent domain types to be transferred to the json schema and adds all the enums to the rust schema which seemed unnecessary. We probably need to look into this further either way since rust throws an exception if it gets an unknown enum which is suboptimal.